examples/simple-filter-xml/main.go

```diff
@@ -47,7 +47,21 @@ func main() {
 		edgexSdk.TransformToXML(),
 		printXMLToConsole,
 	)
-	// 4) Lastly, we'll go ahead and tell the SDK to "start" and begin listening for events
+
+	// 4) shows how to access the application's specific configuration settings.
+	appSettings:= edgexSdk.ApplicationSettings()
+	if appSettings != nil {
+		appName, ok := appSettings["ApplicationName"]
+		if ok {
+			edgexSdk.LoggingClient.Info(fmt.Sprintf("%s now running...", appName))
+		} else {
+			edgexSdk.LoggingClient.Error("ApplicationName application settings not found")
```

Comment:
Should this `return`?

Reply:
For the example I didn't think it was necessary to return so that the example will still run.

Reply:
Adding exit